### ISSUE TITLE
Block stats.lineageos.org

### ISSUE THREAD

**Issue by celenityy** [badmojr/1Hosts#1771]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

<br>

**Domains:**
```
stats.lineageos.org
```
<br>

**Details:**

Used for telemetry in LineageOS, [here](https://www.kuketz-blog.de/lineageos-weder-sicher-noch-datenschutzfreundlich-custom-roms-teil4/)'s a source from a privacy & security expert confirming this is the case.

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>

**Comment by spirillen** [comment]
Danke, it's added to My Privacy DNS project like your other contributions

**Comment by programminghoch10** [comment]
I don't think this is necessary, as the telemetry performed on this domain can be disabled entirely in LineageOS settings.

**Comment by spirillen** [comment]
@programminghoch10 

Allowing tracking is and shall always be a opt-**in** not a opt-out and as long it stays a opt-out, it will remain in my lists for privacy and it is violating humans rights to freedom

**Comment by spirillen** [comment]
I can see in the [history](https://github.com/mypdns/matrix/commit/5fb0cf0e528483c0a6c31662123f8942b37f773c) (git blame)

That the blacklist record is backed up by https://www.kuketz-blog.de/lineageos-weder-sicher-noch-datenschutzfreundlich-custom-roms-teil4/

**Comment by b4dmojr** [comment]
It's the endpoint they use for global device/usage stats.

**Comment by spirillen** [comment]
All telemetry data can be abuse for PII data building as no two devices ever will become equal.

The next question might be, how much data are collected, how detailed is the data collection?

But the fact is, it is collecting user data without asking for permission to do so, which actually are in violation of the GDPR and a hole bunch of other laws


> ## Right to Privacy
> The right to privacy is enshrined in various international human rights instruments, including Article 12 of the Universal Declaration of Human Rights (UDHR) and Article 17 of the International Covenant on Civil and Political Rights (ICCPR). This right encompasses the protection of individuals against arbitrary interference with their privacy, family, home, or correspondence. In the context of the online world, this translates to the protection of personal data and the freedom to communicate without surveillance.
> 
> - https://kb.mypdns.org/articles/MPDNS-A-2

**Comment by b4dmojr** [comment]
Unless there is evidence otherwise (of PII collecting), this will be left to individual users to block at their discretion.

**Comment by celenityy** [comment]
> It's the endpoint they use for global device/usage stats. As far as I konow, no PII is transmitted.

Do you also plan on unblocking Mozilla telemetry domains *(ex. [`incoming-telemetry.thunderbird.net`](incoming-telemetry.thunderbird.net) & [`telemetry.mozilla.org`](https://rethinkdns.com/search?q=telemetry.mozilla.org))*, as well as other privacy-respecting, optional telemetry then? I just feel like there's no consistency here.

**Comment by b4dmojr** [comment]
No. I'll leave it to the individual user to include if they so prefer.

**Comment by celenityy** [comment]
> No. I'll leave it to the individual user to include if they so prefer.

I just don't understand the rationale behind blocking `incoming-telemetry.thunderbird.net` & `telemetry.mozilla.org`, but **not** blocking `stats.lineageos.org`. If your position is to not to block privacy-respecting telemetry, I think that's completely understandable - but you should be consistent about it.
